tst/EnergyPlus/unit/Autosizing/BaseClassSizing.unit.cc

```diff
@@ -59,6 +59,8 @@
 #include <EnergyPlus/DataAirSystems.hh>
 #include <EnergyPlus/DataEnvironment.hh>
 #include <EnergyPlus/DataHVACGlobals.hh>
+#include <EnergyPlus/DataHeatBalSurface.hh>
```

Comment:
Again, no need to introduce two namescopes for `SurfWinCoeffAdjRatioIn` and `SurfWinCoeffAdjRatioOut`.

Reply:
I unified them to DataHeatBalSurface